docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -43,13 +43,23 @@ The mini-application's `Environment` is used to enumerate property sources and p
 The HTTP service has resources in the following form:
 
 ----
-/{application}/{profile}[/{label}]
+/{application}/[{profile}]/{label}
 /{application}-{profile}.yml
 /{label}/{application}-{profile}.yml
 /{application}-{profile}.properties
 /{label}/{application}-{profile}.properties
 ----
 
+For example:
+
+----
+curl localhost:8888/foo/development/master
```

Comment:
Add one that is just /foo/development

Reply:
Our writers discourage "foo", replace it with a real word

Reply:
- agreed with `/foo/development`, I like that.
 - but `foo` is something that is present above in the documentation anyway : `curl localhost:8888/foo/development`; I did not add it myself in there. Also the files in the `https://github.com/spring-cloud-samples/config-repo` are `foo` or `foo-XXX` anyway.

Reply:
yup